### PR TITLE
feat(assets): add ct logos

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: application-templates/starter/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://commercetools.com/">
-    <img alt="commercetools logo" src="http://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20chrom%20black%20horizontal%20RGB%2072dpi.png">
+    <img alt="commercetools logo" src="https://unpkg.com/@commercetools-frontend/assets/logos/commercetools_primary-logo_horizontal_RGB.png">
```
Comment (montezume) [inline application-templates/starter/README.md]: don't we need to do a release first for this to work?

Comment (emmenko) [reply application-templates/starter/README.md]: Yes, but once released it will "just work" 😉

Comment (montezume) [reply application-templates/starter/README.md]: It will just work but we need to update the latest tag

Comment (emmenko) [reply application-templates/starter/README.md]: Yes, I meant to do this today. Have faith my young padawan 😁

**Pull request merged**